components/ContestFilter.tsx

```diff
@@ -11,18 +11,28 @@ export default function ContestFilter({
   statusFilter,
   setStatusFilter,
 }: ContestFilterProps) {
-  const handleStatusFilter = (status: IFilter) => {
-    setStatusFilter(status)
+  const filterMapping: { [key: string]: IFilter } = {
+    All: 'All',
```

Comment:
why is `All` uppercase?

Reply:
because that value is not being returned by the backend, it might as well be in uppercase.